Lombiq.Tests.UI.Samples/Lombiq.Tests.UI.Samples.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>false</IsPackable>
+    <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
```

Comment:
Why is this needed?

Reply:
["Publish error: Found multiple publish output files with the same relative path"](https://github.com/Lombiq/UI-Testing-Toolbox/pull/132/commits/bbb18b4a294b6dd02fd11ac5f75397916d949f64)

Reply:
Where is that happening though? In CI (because I can't see it)?

Reply:
While trying to fix [this](https://ci.lombiq.com/buildConfiguration/OrchardExtensions_OSOCE_Developer_PullAndBuild/950044?showRootCauses=false&expandBuildChangesSection=true&expandBuildProblemsSection=true). I reproduced it locally, fixed it and then I got the above error when rerunning the same command.

Reply:
Didn't you fix this centrally, @BenedekFarkas?

Reply:
Locally you need it of course, but in TeamCity, this parameter is passed into every `dotnet publish` command we run (regardless of calling from PS or using the .NET Runner).

Reply:
OK, so this is not something that would cause any issues in TC. It won't hurt locally though but Dávid please document why this is necessary with this page linked.